ush/templates/FV3LAM_wflow.xml

```diff
@@ -967,7 +969,7 @@ MODULES_RUN_TASK_FP script.
       <and>
         <or>
           <taskdep task="&PROCESS_RADAR_REF_TN;_#type#"/>
-          <taskdep task="&PROCESS_LIGHTNING_TN;_#type#"/>
```

Comment:
Why was the dependency on the lightning step removed?

Reply:
This is not the lightning step used by current workflow. 
The removed one is to process NLDN NetCDF lightning files; but current workflow uses the BUFR lightning files. I corrected the dependency to process_bufr.